linux/RNGEntropy.c

```diff
@@ -0,0 +1,92 @@
+/*
+htop - RNGEntropy.c
+(C) 2022 Loic Jourdheuil
+Released under the GNU GPLv2+, see the COPYING file
+in the source distribution for its full text.
+*/
+
+#include "linux/RNGEntropy.h"
+
+#include <stdlib.h>
+
+#include "CRT.h"
+#include "Object.h"
+#include "XUtils.h"
+
+#define INVALID_VALUE ((unsigned int)-1)
+
+static unsigned int entropy = INVALID_VALUE;
+static unsigned int poolsize = INVALID_VALUE;
```

Comment:
```suggestion
static unsigned size_t = INVALID_VALUE;
static unsigned size_t = INVALID_VALUE;
```

Requires `<stddef.h>`